src/__flowtests__/supress_comment.js

```diff
@@ -6,145 +6,5 @@
 // These examples should suppress the errors correctly.
 // ===
 
-// $FlowAllowDynamicImport
+// $FlowExpectedError[incompatible-type]
 const kw1_1: string = 123;
```

Comment:
Not sure what else to do with this file @mrtnzlml 

We could add other tests than `incompatible-type` maybe?

Reply:
@tbergq Why did you remove the `suppress_comment` settings? It's not configurable anymore?

Reply:
Yeah, they have removed it, flow gives you error saying it is no longer supported

Reply:
> Not sure what else to do with this file @mrtnzlml

I'd remove this file since it was testing `suppress_comment` config but that's no longer configurable.

Reply:
Deleted